### PR TITLE
Implementing Android

### LINKED ISSUE #2: Support Android devices
Add support for Android apps

### PR DESCRIPTION
**Pull request by eyal-sasson**
WIP: resolves #2.
I'm using [JSch](http://www.jcraft.com/jsch/) for SSH connection in Java.

### CODE REVIEW

**Comment by eyal-sasson** [comment]
Just realized we're not completely done as the resizing doesn't work yet

**Comment by eyal-sasson** [comment]
Also see tuzig/terminal7/pull/326

File: android/src/main/java/dev/terminal7/plugins/ssh/SSHPlugin.java [hunk 1]
```diff
@@ -1,22 +1,253 @@
...
+            return;
+        }
+        String command = call.getString("command", "");
+        try {
+            call.setKeepAlive(true);
+            channel.startShell();
+            if (command.length() > 0) {
+                channel.write(command + "\n");
+            }
+            Thread.sleep(2000);
```
Comment (daonb) [inline android/src/main/java/dev/terminal7/plugins/ssh/SSHPlugin.java]: Does it have to be 2 seconds? Aren't we risking losing data?

**Review by daonb — COMMENTED** [review]
Clean simple code. I Found only one Sleep I don't like.

**Pull request merged**